packages/next/src/middleware/sitemaps/createSitemaps.ts

```diff
@@ -1,3 +1,4 @@
+import { stripIndent } from 'common-tags';
```

Comment:
I this tree-shaked? I feel that you could just extract a simple function to trim the extra indentation here instead of importing a library:

https://github.com/zspecza/common-tags/blob/d3f704255f636641e80cefece3ca1c16eb773e71/src/stripIndentTransformer/stripIndentTransformer.js#L8-L31

Reply:
@theodesp Yes! This will indeed be tree shaken. Next.js middleware operates in an [ES modules only runtime](https://nextjs.org/docs/api-reference/edge-runtime#unsupported-apis) with webpack, so only imports that are explicitly used will be bundled.